src/views/OfferInfoPage/components/CheckOfferPage.tsx

```diff
@@ -17,8 +21,15 @@ const CheckOffer = () => {
     navigate(-1);
   };
 
+  //모달창 열림
+  const [isModal, setIsModal] = useState(false);
+  const handleModalOpen = () => {
+    setIsModal(true);
+  };
```

Comment:
```suggestion
  const handleModalToggle = () => {
    setIsModal(!isModal);
  };
```
이렇게 작성하면 모달을 켜는 동작 뿐만 아니라 끄는 동작도 해결할 수 있습니다!